hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/request/bucket/TestOMBucketCreateRequest.java

```diff
@@ -478,4 +506,25 @@ public static void addCreateVolumeToTable(String volumeName,
             .setOwnerName(UUID.randomUUID().toString()).build();
     OMRequestTestUtils.addVolumeToOM(omMetadataManager, omVolumeArgs);
   }
+
+  protected OMBucketCreateRequest doPreExecute(String volumeName,
+      String bucketName,
+      OzoneManagerProtocolProtos.BucketLayoutProto layout) throws Exception {
+    OzoneManagerProtocolProtos.BucketInfo.Builder bucketInfo =
+        newBucketInfoBuilder(bucketName, volumeName)
+            .setBucketLayout(layout);
+    if (layout == OzoneManagerProtocolProtos.BucketLayoutProto.FILE_SYSTEM_OPTIMIZED) {
+      bucketInfo.addMetadata(OMRequestTestUtils.fsoMetadata());
+    }
+    return doPreExecute(bucketInfo);
+  }
```

Comment:
I think this extra `doPreExecute` is not necessary. How about we simply use the current `newBucketInfoBuilder#setBucketLayout(FILE_SYSTEM_OPTIMIZED)`? Also the `addMetadata` is also not necessary.